torchci/lib/bot/verifyDisableTestIssueBot.ts

```diff
@@ -139,6 +142,36 @@ export function formValidationComment(
   return validationCommentStart + body + validationCommentEnd;
 }
 
+export async function formJobValidationComment(
+  context: any,
+  username: string,
+  jobName: string
+): Promise<string> {
+  const authorized = await hasWritePermissions(context, username);
+
+  let body =
+    "<body>Hello there! From the DISABLED prefix in this issue title, ";
+  body += "it looks like you are attempting to disable a job in PyTorch CI. ";
+  body += "The information I have parsed is below:\n\n";
+  body += `* Job name: \`${jobName}\`\n`;
+  body += `* Credential: \`${username}\`\n\n`;
+
+  if (!authorized) {
+    body += `<b>ERROR!</b> Your credential (${username}) does not have the permission to disable ${jobName}.\n\n`;
```

Comment:
nit: making it more personable

```suggestion
    body += `<b>ERROR!</b> You (${username}) don't have permission to disable ${jobName}.\n\n`;
```